hr_holidays_settings/views/res_config_settings.xml

```diff
@@ -30,9 +30,8 @@
           action="hr_holidays_config_settings_action"
           groups="base.group_system"/>
 
-    <menuitem id="menu_hr_holidays_status" name="Leave Types"
-          parent="hr_holidays.menu_hr_holidays_configuration" sequence="10"
-          action="hr_holidays.open_view_holiday_status"
-          groups="base.group_system"/>
+    <record model="ir.ui.menu" id="hr_holidays.hr_holidays_status_menu_configuration">
```

Comment:
I would rather remove completely this "Leave Types" code from here and keep only the standard behavior.

Reply:
That would be behaviour change and frankly speaking I have no preference over this. Yet not a fix totally, that should be addressed as separate PR

Reply:
In V11 the "Configuration" menuitem was opening the "Leave Types" directly (the explicit "Leave Types" menuitem was missing). That's why this module was overriding the two menuitems.

But now in V12, a dedicated "Leave Types" menuitem is present and the "Configuration" menuitem is just a parent of it. So this block of code is now obsolete. Is just a leftover from the migration. The correct fix is to remove it, not changing its group (there's no need to change its group). Keeping the standard behavior is fine.

Reply:
If you say so :) Thanks for details :)